docs/modules/secret-operator/pages/secretclass.adoc

```diff
@@ -125,6 +129,12 @@ spec:
 `autoTls.ca.keyGeneration`:: Configures how keys should be generated.
 `autoTls.ca.keyGeneration.rsa`:: Declares that keys should be generated using the RSA algorithm.
 `autoTls.ca.keyGeneration.rsa.length`:: The amount of bits used for generating the RSA key pair. Currently, `2048`, `3072` and `4096` are supported. Defaults to `2048` bits.
+`additionalTrustRoots`:: Configures additional trust roots which are added to the CA files or truststores in the provisioned volume mounts.
+`additionalTrustRoots.secret`::
```

Comment:
I think these are supposed to be:

~~~suggestion
`autoTls.additionalTrustRoots`:: Configures additional trust roots which are added to the CA files or truststores in the provisioned volume mounts.
`autoTls.additionalTrustRoots.secret`::
~~~

Reply:
Thanks! Fixed in #578